### PR TITLE
Implement --version option

### LINKED ISSUE #699: Let command line 'ert' expose its version
`ert --version` or `ert --help` should reveal the version of ert

### PR DESCRIPTION
**Pull request by hnformentin**
**Issue**
Resolves #699

### CODE REVIEW

**Comment by hnformentin** [comment]
> LGTM! Good job

File: tests/all/test_main.py [hunk 1]
```diff
@@ -162,3 +164,36 @@ def test_verbose_flag(flag_val, expected_value):
...
+
+def test_version_mocked(capsys, monkeypatch):
+    monkeypatch.setattr(ert_shared, "__version__", "1.0.3")
+
+    try:
+        ert_parser(None, ["--version"])
+    except SystemExit as e:
+        assert e.code == 0
+
+    if sys.version_info.major < 3:
```
Comment (jondequinor) [inline tests/all/test_main.py:193]: Same as above, use skip decorator.

File: tests/all/test_main.py [hunk 2]
```diff
@@ -162,3 +164,36 @@ def test_verbose_flag(flag_val, expected_value):
...
+
+
+def test_version_valid_Version(capsys):
+
+    try:
+        ert_parser(None, ["--version"])
+    except SystemExit as e:
+        assert e.code == 0
+
+    if sys.version_info.major < 3:
```
Comment (jondequinor) [inline tests/all/test_main.py:176]: I'm not quite sure why the test returns here? If this isn't supposed to be tested in python 2, then you should use a `@skip` decorator on the test. See https://github.com/equinor/ert/blob/master/tests/all/plugins/test_plugin_context.py#L28

**Comment by jondequinor** [comment]
A bit late to the party, just ignore my comments.

File: tests/all/test_main.py (hunk 2 above)
Comment (sondreso) [reply tests/all/test_main.py:176]: It's just the `capsys` decorator that has a bug in python 2, but the pytest developers have chosen not to fix it :man_shrugging: So the test of the exit code works, it's just that it does not capture stderr.

**Pull request merged**